README.md

```diff
@@ -19,10 +19,11 @@ Note: This does not take into account our dependencies dependencies.
 
 ```bash
 export GITHUB_PAT="<<<the-PAT-you-just-created-above>>"
+export REPO_OWNER="usually your githubid however, it likely isn't the githubID you are using for these notifications."
```

Comment:
maybe "the github id that will be committing update changes in dependent repositories"